class/defaults.yml

```diff
@@ -39,6 +39,23 @@ parameters:
 
     namespace: syn-${_instance}
     name: keycloak
+    images:
```

Comment:
Please be aware that you'll get Renovate PRs for the container images when you specify them like this. If that's the intention, that's fine, but if you want to closely couple the Helm chart version and the container image versions, I'm not sure using this structure is the right choice here.

Reply:
I'm aware off and it is intended. However I probably should add min and max version checks as there are combination are known not to work.

Reply:
See #208.